processor/error/package_tests/json_schema_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"testing"
 
 	er "github.com/elastic/apm-server/processor/error"
+	generatedschemas "github.com/elastic/apm-server/processor/error/generated-schemas"
```

Comment:
Pleas use `schema` over `schemas` - as [Rakyll's post](https://rakyll.org/style-packages/) points out package names should not be plural but singular. 

I am not a huge fan of having `generated-schemas` in the path. The package name is `schemas`, the import path ends with `generated-schemas` and it needs to be overwritten with `generatedschemas` - if not absolutely necessary I'd rather avoid that.
How about either 
* getting rid of `generated` and just use `schema` 
* or use `generated/schema` folder structure to have a cleaner import. This would also allow to add further generated files in the future if necessary.